server/controllers/deleteUser.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Create users module
+ * @module registerUser
+ */
+
+const logger = require('../../tools/logger');
+const database = require('../models/database');
+const validator = require('../utils/validator');
+const constants = require('../utils/constants');
+
+/**
+ * Create a new user API
+ * Get users info and create a new user on the database
+ *
+ * @param {string} req.bo dy.id - User to be created id
+ * @param {string} req.body.battleTag - User to be created battletag
+ * @return {json} - Returns a success message to the user
+ * @throws {json} - Throws a title and body with the error info
+ * 
+ */
+
+//はじめまして！
+module.exports = (req, res) => {
+  let { id } = req.body;
+  id = parseInt(id, 10);
+  console.log("id: " + id);
+  
+  if (!validator.isValidInteger(id))
+    return res.status(400).json({
+      msg: constants.messages.error.INVALID_ID
+    });
+
+  database.user.destroy({
+    where: {
+      id: id 
+    }
+  }).then(function(rowDeleted){
+    if(rowDeleted === 1){
+      return res.status(201).json({
+        msg: 'Your id "' + id + '" was successfully found and exterminated!'
+      });
+    } else {
+      return res.status(400).json({
+        msg: 'User not found.'
```

Comment:
Essa mensagem deveria estar dentro de constants, olha com as outras estão

Reply:
Vdd faz sentido. Na minha cabeça isso seria um erro que só esse retornaria, mas consigo usar dps pro GET BTAG ne

Reply:
Consegue usar em outros lugares. Tipo se a gente cria uma API que recebe um telegram id, ve o usuário e retorna uns status dele sei lá. Teria essa validação. String fixa assim 99% das vezes tem que ir pra constants

Reply:
mesmo se for só nesse é bom estar nas constants, pra centralizar tudo, se quiser mudar qualquer mensagem da aplicaçao já sabe onde tem que ir, e nunca se sabe quando vc pode utilizar em outros lugares tbm

Reply:
vdd

Reply:
a bolicia ensinando ao invez de repreendendo, exemplo de pessoa